drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt

```diff
@@ -345,5 +346,22 @@ class DropInViewModel(
                 putExtra(DROP_IN_RESULT_INTENT_KEY, resultHandlerIntent)
             }
         }
+
+        fun assertBundleExists(bundle: Bundle?): Boolean {
+            if (bundle == null) {
```

Comment:
I see the check is failing because of the returnCount, you can probably avoid it by doing something like:
```Kotlin
return when {
    bundle == null -> { ... }
    !bundle.containsKey(PAYMENT_METHODS_RESPONSE_KEY) ||
    !bundle.containsKey(DROP_IN_CONFIGURATION_KEY) ||
    !bundle.containsKey(DROP_IN_RESULT_INTENT_KEY) -> { ... }
    else -> true
}
```

Reply:
👍